.github/workflows/causal_lm_cpp.yml

```
@@ -524,3 +524,58 @@ jobs:
           && export PYTHONPATH=./build/:$PYTHONPATH
           && timeout 50s samples/python/greedy_causal_lm/greedy_causal_lm.py ./redpajama-3b-chat/ "Alan Turing was a"
           | diff ./pred_greedy.txt -
+
+  cpp-chat_sample-ubuntu:
+    runs-on: ubuntu-20.04-16-cores
```

```suggestion
    runs-on: ubuntu-20.04
```
Tiny llama usually fits the cheaper runner.

---

done